snc-library.sh

```diff
@@ -242,14 +242,63 @@ function renew_certificates() {
 
 # deletes an operator and wait until the resources it manages are gone.
 function delete_operator() {
-        local delete_object=$1
-        local namespace=$2
-        local pod_selector=$3
+    local delete_object=$1
+    local namespace=$2
+    local pod_selector=$3
 
-	retry ${OC} get pods
-        pod=$(${OC} get pod -l ${pod_selector} -o jsonpath="{.items[0].metadata.name}" -n ${namespace})
+    retry ${OC} get pods
+    pod=$(${OC} get pod -l ${pod_selector} -o jsonpath="{.items[0].metadata.name}" -n ${namespace})
 
-        retry ${OC} delete ${delete_object} -n ${namespace}
-        # Wait until the operator pod is deleted before trying to delete the resources it manages
-        ${OC} wait --for=delete pod/${pod} --timeout=120s -n ${namespace} || ${OC} delete pod/${pod} --grace-period=0 --force -n ${namespace} || true
+    retry ${OC} delete ${delete_object} -n ${namespace}
+    # Wait until the operator pod is deleted before trying to delete the resources it manages
+    ${OC} wait --for=delete pod/${pod} --timeout=120s -n ${namespace} || ${OC} delete pod/${pod} --grace-period=0 --force -n ${namespace} || true
 }
+
+function all_operators_available() {
+    # Check the cluster operator output, status for available should be true for all operators
```

Comment:
might be easier to understand that code in the future if you add after your comment "so the single line `oc get co` output should not contain any occurrences of `False"?